lib/data_hygiene/organisation_reslugger.rb

```diff
@@ -59,8 +59,6 @@ def new_base_path
       case organisation
       when Organisation
         new_slug.public_path
-      when WorldwideOrganisation
```

Comment:
it looks like this `new_base_path` method hasn't been used for a long time, so the whole thing could go if you're in a deleting kinda mood

Reply:
Good spot.  I've removed this unused method.